scripts/aks/bootstrap.sh

```diff
@@ -367,6 +367,47 @@ else
     echo "    NSG exists."
 fi
 
+# NSG Flow Logs
+FLOW_LOGS_STORAGEACCOUNT_EXIST=$(az storage account list --resource-group "$AZ_RESOURCE_GROUP_LOGS" --query "[0].name=='$AZ_RESOURCE_STORAGEACCOUNT_FLOW_LOGS'")
+FLOW_LOGS_STORAGEACCOUNT_ID="/subscriptions/$AZ_SUBSCRIPTION_ID/resourceGroups/$AZ_RESOURCE_GROUP_LOGS/providers/Microsoft.Storage/storageAccounts/$AZ_RESOURCE_STORAGEACCOUNT_FLOW_LOGS"
+
+if [ ! "$FLOW_LOGS_STORAGEACCOUNT_EXIST" ]; then
+    printf "Flow logs storage account does not exists.\n"
+
+    printf "    Creating storage account %s" "$AZ_RESOURCE_STORAGEACCOUNT_FLOW_LOGS"
+        az storage account create \
+        --name "$AZ_RESOURCE_STORAGEACCOUNT_FLOW_LOGS" \
+        --resource-group "$AZ_RESOURCE_GROUP_LOGS" \
+        --location "$AZ_RADIX_ZONE_LOCATION" \
+        --sku "Standard_LRS" \
+        --subscription "$AZ_SUBSCRIPTION_ID"
+    printf "Done.\n"
+else
+    printf "    Storage account exists.\n"
+fi
+
+if [ "$FLOW_LOGS_STORAGEACCOUNT_EXIST" ]; then
+    NSG_FLOW_LOGS="$(az network nsg show -g "$AZ_RESOURCE_GROUP_CLUSTERS" -n "$NSG_NAME" | jq -r .flowLogs)"
+
+    # Check if NSG has assigned Flow log
+    if [[ $NSG_FLOW_LOGS != "null" ]]; then
+        printf "    There is an existing Flow Log on %s\n" "$NSG_NAME"
+    else
+        # Create network watcher flow log and assign to NSG
+        printf "    Creating azure Flow-log %s..." "${NSG_NAME}-rule"
+        az network watcher flow-log create \
+            --name "${NSG_NAME}-flow-log" \
+            --resource-group "$AZ_RESOURCE_GROUP_CLUSTERS" \
+            --nsg "$NSG_NAME" \
+            --location "$AZ_RADIX_ZONE_LOCATION" \
+            --storage-account "$FLOW_LOGS_STORAGEACCOUNT_ID" \
+            --subscription "$AZ_SUBSCRIPTION_ID" \
+            --retention "90" \
+            --enable true
```

Comment:
Set `--output none` to exclude the json object from the output.
```suggestion
            --enable true \
            --output none
```